types/vector/fundamental/README.md

```diff
@@ -0,0 +1,13 @@
+# `std::vector<std::int32_t>`
+
+## Fields
+
+ * `[Split]Index{32,64}`
+
+with the corresponding column type for the collection parent field.
+
+## Entries
+
+1. One element vectors, with ascending values
+2. Empty vectors
+3. Increasing number of elements per vector
```

Comment:
I would maybe specify here that you're increasing them across the four fields, rather than multiple entries with increasing vector lengths.